### PR TITLE
Add Ukrainian translation

### PR DESCRIPTION
**Pull request by Higmans**
I translated the page to Ukrainian language

### CODE REVIEW

File: uk/index.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (tulir) [inline uk/index.html:2]:
```suggestion
<html lang="uk">
```

File: uk/index.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+<head>
+    <title>Не питайте аби спитати, просто спитайте</title>
+    <meta property="og:title" content="Не питайте аби спитати, просто спитайте" />
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://dontasktoask.com" />
```
Comment (tulir) [inline uk/index.html:7]:
```suggestion
    <meta property="og:url" content="https://dontasktoask.com/uk/" />
```

File: uk/index.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+        Інші подібні проблеми: <a href="https://xyproblem.info/">The XY Problem</a>, <a href="https://nohello.net/uk">Ні «Привіт»</a>.
+        Більше інформації: <a href="https://stackoverflow.com/help/how-to-ask">How do I ask a good question?</a>,
+        або якщо маєте більше часу: <a href="http://catb.org/~esr/faqs/smart-questions.html">How To Ask Questions The Smart Way</a>.
+    </p>
+</main>
+<footer>
+    Контент здебільшого вкрадено з <a href="https://iki.fi/sol/dontask.html">iki.fi/sol/dontask.html</a>
+    -
+    <a href="https://github.com/maunium/dontasktoask.com">Програмний код на GitHub</a>
+</footer>
```
Comment (tulir) [inline uk/index.html:88]: You can add translation credits to the footer and/or readme if you want

**Review by Wirtos — APPROVED** [review]
Translation seems correct so far

**Review by kittyandrew — CHANGES_REQUESTED** [review]

File: uk/index.html [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+        <span class="name">Foobar123:</span>
+        <p class="message">
+            Чи є тут експерти з Java, які готові приділити час на вирішення моєї
+            проблеми, будь-якою б вона не була, навіть якщо вона насправді не
+            пов'язана з Java або на неї може відповісти хтось, хто не має жодного
+            досвіду з Java?
+        </p>
+    </blockquote>
+    <p>
+        Існує багато причин, чому люди, які МАЮТЬ знання, можуть не визнавати цього.
```
[the remark this answers is not in the document]
Comment (Higmans) [reply uk/index.html]: Fixed, thanks!

**Review by kittyandrew — APPROVED** [review]

**Review by tulir — APPROVED** [review]

**Pull request merged**